### PR TITLE
Update labeler.yml

### PR DESCRIPTION
**Pull request by dvir001**
Adding missing regex

### CODE REVIEW

**Review by MagnusCrowe — APPROVED** [review]

File: .github/labeler.yml [hunk 1]
```diff
@@ -17,10 +20,14 @@
...
  - "Resources/Maps/_NF/Shuttles/*.yml" # Grid Files
  - "Resources/Prototypes/_NF/Shipyard/*.yml" # Prototypes Files
+ - "Resources/Maps/_NF/Shuttles/*/*.yml" # Grid Files
+ - "Resources/Prototypes/_NF/Shipyard/*/*.yml" # Prototypes Files
 
 "Map-POI":
  - "Resources/Maps/_NF/POI/*.yml" # Grid Files
  - "Resources/Prototypes/_MF/Maps/POI/*.yml" # Prototypes Files
+ - "Resources/Maps/_NF/POI/*/*.yml" # Grid Files
+ - "Resources/Prototypes/_MF/Maps/POI/*/*.yml" # Prototypes Files
```
Comment (whatston3) [inline .github/labeler.yml:30]: Change to **?

```suggestion
 - "Resources/Maps/_NF/POI/**/*.yml" # Grid Files
 - "Resources/Prototypes/_MF/Maps/POI/**/*.yml" # Prototypes Files
```

File: .github/labeler.yml [hunk 2]
```diff
@@ -17,10 +20,14 @@
 "Map-Shuttle":
  - "Resources/Maps/_NF/Shuttles/*.yml" # Grid Files
  - "Resources/Prototypes/_NF/Shipyard/*.yml" # Prototypes Files
+ - "Resources/Maps/_NF/Shuttles/*/*.yml" # Grid Files
+ - "Resources/Prototypes/_NF/Shipyard/*/*.yml" # Prototypes Files
```
Comment (whatston3) [inline .github/labeler.yml:24]: Change to **?
```suggestion
 - "Resources/Maps/_NF/Shuttles/**/*.yml" # Grid Files
 - "Resources/Prototypes/_NF/Shipyard/**/*.yml" # Prototypes Files
```

File: .github/labeler.yml [hunk 3]
```diff
@@ -3,12 +3,15 @@
...
  - "Resources/Maps/_NF/Admin/*.yml" # Grid Files
+ - "Resources/Maps/_NF/Admin/*/*.yml" # Grid Files
 
 "Map-Bluespace":
  - "Resources/Maps/_NF/Bluespace/*.yml" # Grid Files
+ - "Resources/Maps/_NF/Bluespace/*/*.yml" # Grid Files
 
 "Map-Dungeon":
  - "Resources/Maps/_NF/Dungeon/*.yml" # Grid Files
+ - "Resources/Maps/_NF/Dungeon/*/*.yml" # Grid Files
```
Comment (whatston3) [inline .github/labeler.yml:14]: Change to **?

```suggestion
 - "Resources/Maps/_NF/Dungeon/**/*.yml" # Grid Files
```

File: .github/labeler.yml [hunk 4]
```diff
@@ -3,12 +3,15 @@
 
 "Map-Admin":
  - "Resources/Maps/_NF/Admin/*.yml" # Grid Files
+ - "Resources/Maps/_NF/Admin/*/*.yml" # Grid Files
 
 "Map-Bluespace":
  - "Resources/Maps/_NF/Bluespace/*.yml" # Grid Files
+ - "Resources/Maps/_NF/Bluespace/*/*.yml" # Grid Files
```
Comment (whatston3) [inline .github/labeler.yml:10]: Change to **?
```suggestion
 - "Resources/Maps/_NF/Bluespace/**/*.yml" # Grid Files
```

File: .github/labeler.yml [hunk 5]
```diff
@@ -3,12 +3,15 @@
 
 "Map-Admin":
  - "Resources/Maps/_NF/Admin/*.yml" # Grid Files
+ - "Resources/Maps/_NF/Admin/*/*.yml" # Grid Files
```
Comment (whatston3) [inline .github/labeler.yml:6]: Change to **?
```suggestion
 - "Resources/Maps/_NF/Admin/**/*.yml" # Grid Files
```

**Review by whatston3 — COMMENTED** [review]
For all of the entries here, is there a reason to avoid use of double stars (**)? As far as I understand, this would include both the base folder and all subfolders.

File: .github/labeler.yml (hunk 5 above)
Comment (dvir001) [reply .github/labeler.yml:6]: If a file is Resources/Maps/_NF/Admin/test.ftl will it be marked as a map if i change this?

**Pull request merged**